server/container/guice/pom.xml

```diff
@@ -37,21 +37,8 @@
 
     <modules>
         <module>guice-common</module>
+        <module>cassandra-guice</module>
+        <module>memory-guice</module>
     </modules>
-
-    <profiles>
-        <profile>
```

Comment:
I think you can just activate cassandra smtp tests if cassandra profile is present. Right ?